Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
I'll just wait for UNIX builds to stabilize, then.
I do think that "large" PRs are inevitable in some cases. We might revisit that issue when I submit one of those in the near future XD